site/versioning-policy.html

```diff
@@ -339,15 +339,15 @@ <h3>Spark 3.1 Release Window</h3>
   </thead>
   <tbody>
     <tr>
-      <td>Early Dec 2020</td>
+      <td>June 1st 2021</td>
       <td>Code freeze. Release branch cut.</td>
     </tr>
     <tr>
-      <td>Mid Dec 2020</td>
+      <td>Mid June 2021</td>
       <td>QA period. Focus on bug fixes, tests, stability and docs. Generally, no new features merged.</td>
     </tr>
     <tr>
-      <td>Early Jan 2021</td>
+      <td>July 2021</td>
```

Comment:
We released 3.1.1. on March 2, 2021. Based on the 6 month release cycle, the target release date should be Sept 2nd. This does not sound right to me.

Reply:
I'm fixing it in https://github.com/apache/spark-website/pull/331